### ISSUE TITLE
ZG9092

### ISSUE THREAD

**Issue by iFac3** [easyhomesrc/homey_easylighting#22]
Thermostat target temp can’t be adjusted from Homey Pro (early23) interface.

**Comment by maginawin** [comment]
Hi, I'm getting a new Homey Pro for this, I only have a Homey and a Homey Bridge right now.

**Comment by iFac3** [comment]
I do not believe this issue is related only to e23 version of Pro, both e19 versions should be affected too. Please check the scrshot of the error message.

![IMG_53C92F72BA3A-1](https://github.com/easyhomesrc/homey_easylighting/assets/51527573/cc2f92ab-612a-47e0-add7-fe46171fc535)
